proprietary/purmerend-design-tokens/figma/color-scheme-dark.tokens.json

```diff
@@ -230,8 +241,174 @@
       }
     }
   },
+  "components/button": {
+    "utrecht": {
+      "button": {
+        "disabled": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.disabled.interactive-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "{basis.color.transparent}"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.disabled.text-1}"
+          }
+        },
+        "primary-action": {
+          "hover": {
+            "background-color": {
+              "$type": "color",
+              "$value": "{basis.color.info.interactive-1}"
+            },
+            "border-color": {
+              "$type": "color",
+              "$value": "transparent"
+            }
+          }
+        }
+      }
+    }
+  },
+  "components/checkbox": {
+    "utrecht": {
+      "checkbox": {
+        "checked": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{purmerend.color.black-2}"
+          }
+        },
+        "indeterminate": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{purmerend.color.black-2}"
+          }
+        }
+      }
+    }
+  },
+  "components/radio-button": {
+    "utrecht": {
+      "radio-button": {
+        "disabled": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.disabled.bg-1}"
```

Comment:
In plaats van `disabled.bg-1` op dark mode in te stellen voor alle form-componenten in dark mode, zou het niet werken om `basis.form-control.disabled.background-color` in te stellen op purmemerend light mode niveau?